entrypoint.sh

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/env bash
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# Might be empty
+AIRFLOW_COMMAND="${1}"
+
+set -euo pipefail
+
+function verify_db_connection {
+    DB_URL="${1}"
+
+    DB_CHECK_MAX_COUNT=${MAX_DB_CHECK_COUNT:=10}
+    DB_CHECK_SLEEP_TIME=${DB_CHECK_SLEEP_TIME:=2}
+
+    export DETECTED_DB_BACKEND=""
+    export DETECTED_DB_USER=""
+    export DETECTED_DB_HOST=""
+    export DETECTED_DB_PORT=""
+    export DETECTED_DB_SCHEMA=""
+    export DETECTED_DB_PARAMS=""
```

Comment:
```suggestion
    DETECTED_DB_BACKEND=""
    DETECTED_DB_USER=""
    DETECTED_DB_HOST=""
    DETECTED_DB_PORT=""
    DETECTED_DB_SCHEMA=""
    DETECTED_DB_PARAMS=""
```

These aren't used by any subcommands, so no need to export them.

Reply:
True

Reply:
(It doesn't matter/affect anything much, and wouldn't have commented unless there were other changes to make.)

Reply:
:D No worries.

Reply:
BTW. I think the reason I had them exported was because user/schema/params are not used and spellcheck complained. So I quickly exported them but then forgot to solve it properly. I solved it now by changing them to local and leaving comments in place of not used params.